data/mods/Aftershock/player/afs_professions.json

```diff
@@ -33,9 +33,9 @@
     "skills": [ { "level": 5, "name": "computer" } ],
     "items": {
       "both": {
-      "itmes": [ "laptop", "software_useless", "linuxtshirt", "pants_cargo", "socks", "slingpack", "sneakers", "smart_phone" ],
-      "entries": [ { "item": "light_plus_battery_cell", "ammo-item": "battery", "charges": 150, "container-item": "eink_tablet_pc" } ]
-    },
+        "itmes": [ "laptop", "software_useless", "linuxtshirt", "pants_cargo", "socks", "slingpack", "sneakers", "smart_phone" ],
```

Comment:
I imagine it should be "items" not "itmes"?

Reply:
Thanks for catching that!